test/modules/diffusion/diffusion_3d_1_poly_ip.lua

```diff
@@ -1,120 +0,0 @@
-materials = {}
-materials[0] = PhysicsAddMaterial("Test Material");
-
-PhysicsMaterialAddProperty(materials[0],SCALAR_VALUE)
-PhysicsMaterialSetProperty(materials[0],SCALAR_VALUE,SINGLE_VALUE,1.0)
-
---############################################### Setup Physics
-phys1 = DiffusionCreateSolver()
-SolverSetBasicOption(phys1,"discretization_method","PWLD_MIP")
-SolverSetBasicOption(phys1,"residual_tolerance",1.0e-6)
-
---############################################### Set boundary conditions
-DiffusionSetProperty(phys1,"boundary_type","ZMIN","reflecting")
-DiffusionSetProperty(phys1,"boundary_type","ZMAX","reflecting")
-
---############################################### Initialize and Execute Solver
-DiffusionInitialize(phys1)
-DiffusionExecute(phys1)
---
---############################################### Get field functions
-fflist,count = SolverGetFieldFunctionList(phys1)
-
---############################################### Slice plot
-slice1 = FFInterpolationCreate(SLICE)
-
-FFInterpolationSetProperty(slice1,SLICE_POINT,0.008,0.0,0.0)
-FFInterpolationSetProperty(slice1,SLICE_BINORM,0.0,0.0,1.0)
-FFInterpolationSetProperty(slice1,SLICE_TANGENT,0.0,-1.0,0.0)
-FFInterpolationSetProperty(slice1,SLICE_NORMAL,1.0,0.0,0.0)
-FFInterpolationSetProperty(slice1,ADD_FIELDFUNCTION,fflist[1])
-
-FFInterpolationInitialize(slice1)
-FFInterpolationExecute(slice1)
-
-slice2 = FFInterpolationCreate(SLICE)
-FFInterpolationSetProperty(slice2,SLICE_POINT,0.0,0.0,0.024)
-FFInterpolationSetProperty(slice2,ADD_FIELDFUNCTION,fflist[1])
-
-FFInterpolationInitialize(slice2)
-FFInterpolationExecute(slice2)
-
---############################################### Line plot
-line0 = FFInterpolationCreate(LINE)
-FFInterpolationSetProperty(line0,LINE_FIRSTPOINT,-1.0,0.0,0.025)
-FFInterpolationSetProperty(line0,LINE_SECONDPOINT, 1.0,0.0,0.025)
-FFInterpolationSetProperty(line0,LINE_NUMBEROFPOINTS, 100)
-FFInterpolationSetProperty(line0,ADD_FIELDFUNCTION,fflist[1])
-
-FFInterpolationInitialize(line0)
-FFInterpolationExecute(line0)
-
---############################################### Volume integrations
-ffi1 = FFInterpolationCreate(VOLUME)
-curffi = ffi1
-FFInterpolationSetProperty(curffi,OPERATION,OP_MAX)
-FFInterpolationSetProperty(curffi,LOGICAL_VOLUME,vol0)
-FFInterpolationSetProperty(curffi,ADD_FIELDFUNCTION,fflist[1])
-
-FFInterpolationInitialize(curffi)
-FFInterpolationExecute(curffi)
-maxval = FFInterpolationGetValue(curffi)
-
-Log(LOG_0,string.format("Max-value=%.5f", maxval))
-
---############################################### Exports
-if (master_export == nil) then
-    FFInterpolationExportPython(slice1)
-    FFInterpolationExportPython(slice2)
-    FFInterpolationExportPython(line0)
-    ExportFieldFunctionToVTK(fflist[1],"ZPhi3D","Temperature")
-end
-
---############################################### Plots
-if (location_id == 0 and master_export == nil) then
-    local handle = io.popen("python ZPFFI10.py")
-    local handle = io.popen("python ZLFFI20.py")
```

Comment:
why were there 2 calls? Two exports ??? Is there a functionality we need to better document or be aware of in FFexport?